cmd/ethrex/bench/build_block_benchmark.rs

```diff
@@ -182,44 +180,29 @@ async fn fill_mempool(b: &Blockchain, accounts: Vec<SecretKey>) {
     }
 }
 
-pub async fn bench_payload(input: &(&mut Blockchain, Block, &Store)) -> (Duration, u64) {
-    let (b, genesis_block, store) = input;
+pub async fn bench_payload(input: &(Arc<Blockchain>, Block, &Store)) -> (Duration, u64) {
+    let (blockchain, genesis_block, store) = input;
+    let blockchain = std::sync::Arc::new(blockchain);
```

Comment:
isnt this equivalent to `blockchain.clone()`

Reply:
I believe this produces an `Arc<Arc<Blockchain>>` instead. Don't know why would want that tho, cloning the original `Arc<Blockchain>` should be better.